pkgs/development/python-modules/debts/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, python, buildPythonPackage, fetchFromGitLab, jinja2 }:
+
+buildPythonPackage rec {
+  pname = "debts";
+  version = "0.5";
+
+  # pypi does not ship tests
+  src = fetchFromGitLab {
+    domain = "framagit.org";
+    owner = "almet";
+    repo = "debts";
+    rev = "09ee167c"; # no tags
```

Comment:
That's a shame. In that case we should at the very least use a fully-qualified rev instead of a shortened one.

But I kind of think "no tags (-> no automatic updates)" outweighs "no tests", so I'd rather fetch it from pypi for now. Maybe create an issue against the upstream repo, notifying them that tags would be useful :)

Reply:
changed to full revision.